db/dcrpg/internal/blockstmts.go

```diff
@@ -174,6 +174,10 @@ const (
 		WHERE is_valid = FALSE
 		ORDER BY height DESC;`
 
+	SelectTxsPerDay = `SELECT date_trunc('day',time) AS date, sum(numtx)
+		FROM blocks
+		GROUP BY date ORDER BY date;`
```

Comment:
This is the new query on `blocks`, using `sum(numtx)`.

Reply:
This is far much better.